firecracker-control/local.go

```diff
@@ -117,8 +125,12 @@ func (s *local) CreateVM(requestCtx context.Context, req *proto.CreateVMRequest)
 
 	// If we're here, there is no pre-existing shim for this VMID, so we spawn a new one
 	defer shimSocket.Close()
+	if err := os.MkdirAll(s.config.ShimBaseDir, 0644); err != nil {
```

Comment:
I think this should 0700, dirs need the `x` bit set in order to grant search-permissions and I don't see any reason to grant read access to group or other.

Reply:
Fixed!